src/utils/detectionApi.ts

```diff
@@ -359,32 +358,33 @@ class DetectionMapper {
         externalId: change.update.externalId,
       },
     };
-    (eventPatch as EventPatchHack).update.subType = change.update.label;
+    (eventPatch as EventPatchHack).update.subtype = change.update.label;
     if (change.update.metadata) {
```

Comment:
This line could be replaced if 
(change.update.metadata || change.update.box || change.update.fileExternalId || change.update.box) 
(preferably put into function shouldProcessUpdateMetadata(change.update) ) And then only get the mapper if needed. Getting the mapper might cause an sdk retrieve, which we want to avoid unless needed